### PR TITLE
initial french translation

### PR DESCRIPTION
**Pull request by dlucasd**
Hi,

Here's the french translation of this great template.
Thanks!

### CODE REVIEW

**Comment by gernotstarke** [comment]
Hi there,
thanx for your awesome contribution...

We're briefly reviewing it (I just found a French-speaking colleague)...

It would be good if you could reveal your name, so we can add a "thank you" on the landing page,
when we announce the new version!!
It would be perfectly ok if you prefer not to be named, then it will be an anonymous contribution...

regards from Cologne,
Gernot (co-founder and core maintainer of arc42)

**Comment by gernotstarke** [comment]
thanx @martinweck for providing review comments

**Comment by dlucasd** [comment]
My name is Damien Lucas!

Regards from Bordeaux
Damien

File: FR/asciidoc/src/05_building_block_view.adoc
Comment (martinweck) [reply FR/asciidoc/src/05_building_block_view.adoc:26]: I have informed Gernot about it who may adapt it as well

**Review by martinweck — CHANGES_REQUESTED** [review]
Hi Damien,
Thank you so much for your contribution. I am not a native speaker. Hence, if you go through my comments and you feel that they are just wrong, because of my lack of language skills, then just close the comment without action. We have asked also a French-speaking colleague to have a look as well, but for me, it looks all great, only maybe some minor remarks.

File: FR/asciidoc/src/12_glossary.adoc
Comment (martinweck) [inline FR/asciidoc/src/12_glossary.adoc:26]: Maybe, you could add a table here. IMHO it makes it easier for the reader to understand how a glossary could look like, something like:

```
[options="header",cols="1,1"]
|===
|Terme|Définition
| _<Term-1>_ | _<Définition-1>_
| _<Term-2>_ | _<Définition-2>_ | _<Solution-2>_ | _<Lien-2>_
|===
```

**Review by martinweck — APPROVED** [review]
since there are no important things, as with Gernot discussed it's approved

**Review by martinweck — APPROVED** [review]
since there are no important things, as with Gernot discussed it's approved

**Review by martinweck — APPROVED** [review]

**Review by gernotstarke — APPROVED** [review]
Damien, if you like the comments from @martinweck , please feel free to include them.
Otherwise, I gladly accept your PR.

Due to the technical setup of arc42, it might take a few days to have the download available on the official website (we have another person review changes to the downloads... )

Thanx again, merci beaucoup!

File: FR/asciidoc/src/01_introduction_and_goals.adoc
Comment (dlucasd) [reply FR/asciidoc/src/01_introduction_and_goals.adoc:24]: Sure, "des documents relatifs aux exigences" is more indirect. But "des documents d'exigences" is not quite good.
We debated with my colleagues and "des documents décrivant les exigences" is more accurate

File: FR/asciidoc/src/05_building_block_view.adoc
Comment (dlucasd) [reply FR/asciidoc/src/05_building_block_view.adoc:146]: Sure, the third part is redundant. I change that.

**Comment by dlucasd** [comment]
Hi @gernotstarke , @martinweck ,

I've made the corrections in response to the @martinweck's code review.

**Comment by martinweck** [comment]
Hi @dlucasd,
Thanks a lot for those updates! For me, it is perfect now and can be merged.

**Comment by martinweck** [comment]
@gernotstarke github displays "1 workflow awaiting approval", maybe you can look into it and then you can merge it (I think I don't have the rights)

**Review by gernotstarke — APPROVED** [review]
thanx everybody

**Comment by gernotstarke** [comment]
@dlucasd I gave you triage-rights on the repo, so I can assign you as reviewer to Pull-Requests...

Reason: We received a few comments to your french translation - and my French is too bad to comment on those :-)

Would be great if you could accept the triage invite, so I can assign you to this new PR...

**Pull request merged**
